x-pack/plugins/security_solution/cypress/e2e/detection_rules/bulk_edit_rules.cy.ts

```diff
@@ -432,8 +432,8 @@ describe('Detection rules, bulk edit', () => {
       waitForBulkEditActionToFinish({ rulesCount: expectedNumberOfNotMLRules });
 
       cy.get(TOASTER_BODY).should(
-        'have.text',
-        `You've successfully updated ${expectedNumberOfNotMLRules} rules. If you did not select to apply changes to rules using Kibana data views, those rules were not updated and will continue using data views.`
+        'contain.text',
+        `You've successfully updated ${expectedNumberOfNotMLRules} rules.`
```

Comment:
`have.text` matcher helps to detect if the proper message is shown. This way with `contain.text` these two tests are almost identical. I suggest to either revert to `have.text` and check for the special message when some rules were skipped or remove one of the tests.

Reply:
Adressed 👍